src/BeamAdapter/component/mapping/AdaptiveBeamMapping.inl

```diff
@@ -259,23 +269,34 @@ void AdaptiveBeamMapping< TIn, TOut>::applyJ(const core::MechanicalParams* mpara
 
     InVecCoord xBuf2;
 
+    computeDistribution();
```

Comment:
I see line 202 that this method is only call if (d_useCurvAbs.getValue() && !d_contactDuplicate.getValue())
I didn't deep into the code but shouldn't you do the same check?

Reply:
no idea, so I will add it 🤡

Reply:
👨‍🔬